src/snmalloc/backend_helpers/range_helpers.h

```diff
@@ -16,7 +16,7 @@ namespace snmalloc
 
     // Find the minimum set of maximally aligned blocks in this range.
     // Each block's alignment and size are equal.
-    while (length >= sizeof(void*))
+    while (length >= bits::one_at_bit(MIN_BITS))
```

Comment:
I guess we always use this to construct allocatable things at the moment, so there's not much point in going smaller, but I wonder if the lower bound should be a template parameter?

Reply:
`MIN_BITS` is actually a template parameter.  The global constant is `MIN_ALLOC_BITS`.  It was only when I went to change the code, I realised this though :)